### DISCUSSION TITLE
Just meow selection in emacs

### DISCUSSION THREAD

**Discussion by Inc0n** [meow-edit/meow#262] [Ideas]
Evil visual mode is very easy to use. How meow handles selection is also refreshing and powerful.

However, I have found that emacs default binds flows a lot more better than these modal editing packages, but I do miss the how selection can be made in them.

Would it be possible to just meow selection commands without any of the meow states being active?

**Comment by DogLooksGood** [comment]
I had this idea in the past, after I had a stable version of Meow. Adding some selection support in vanilla Emacs instead of buying the whole modal system. For example, when you move forward a word with `M-f`, also select the content you passed. And then press digit keys to quickly expand. It's totally doable, even though has some down sides. But, unfortunately, Meow doesn't have this feature for now.

**Reply by Inc0n** [reply]
I see, would you not think that using an emacs binds in `(meow-define-keys 'normal ...)` may also work, i.e. bind meow-forward to `M-f`, however, leaving out (or disable) all the keys that enable state changes. 
That would be a temporary solution (or integration)

**Reply by DogLooksGood** [reply]
Well, personally, the way Meow works is good enough, so I have no motivation to work on non-modal solution. If I have more time, I probably migrate Meow to other editors as well.

BTW, I'm comfortable to press control/alt with a good keyboard, but not okay with laptop keyboard. Unfortunately, sometimes I have to use a laptop keyboard. But without pressing too much modifiers, laptop keyboard is also good to me.

**Reply by Inc0n** [reply]
I see, for the issue of modifiers, yh I agree this can be a bit of a problem. If I may suggest to you, had you not configured it so already, to check out methods of swapping or setting capslock to control, this one simple change helps immensively.
A step further on this, is to use something similar to `[kmonad](https://github.com/kmonad/kmonad)`, which allows you do much more sophisticated keyboard configurations. However, it is more difficult to setup on mac, therefore, I had not had the opportunity to try it out.
Lastly, even with caps as control, using it requires pressing with pinky, which may cause stress and I often experience with model editings in emacs (namely evil).
